test/src/base/MooseTestApp.C

```diff
@@ -31,7 +31,7 @@ validParams<MooseTestApp>()
                                    "--disallow-test-objects",
                                    false,
                                    "Don't register test objects and syntax");
-  params.set<bool>("automatic_automatic_scaling") = false;
+  params.set<bool>("automatic_automatic_scaling") = true;
```

Comment:
![image](https://user-images.githubusercontent.com/3036960/60897715-ca1e8c00-a225-11e9-8bde-403bd683307a.png)

BTW - How does this work? I didn't see where you are retrieving this parameter. Is it a typo?

Reply:
Oh I see - it's already in the codebase, just not this PR.